src/sparql/update/insert_nord_label.sparql

```diff
@@ -0,0 +1,23 @@
+PREFIX owl: <http://www.w3.org/2002/07/owl#>
+PREFIX oboInOwl: <http://www.geneontology.org/formats/oboInOwl#>
+
+# Add MONDO:NORD_LABEL value as source annotation on synonym.
+
+INSERT {
+  ?axiom oboInOwl:source "MONDO:NORD_LABEL" .
+}
+WHERE {
+  ?class a owl:Class ;
+         oboInOwl:hasExactSynonym ?syn_value .
+
+  ?axiom a owl:Axiom ;
+         owl:annotatedSource ?class ;
+         owl:annotatedProperty oboInOwl:hasExactSynonym ;
+         owl:annotatedTarget ?syn_value ;
+         oboInOwl:hasSynonymType ?syn_type .
+
+  FILTER(STR(?syn_type) = "http://purl.obolibrary.org/obo/mondo#NORD_LABEL")
+  FILTER NOT EXISTS {
```

Comment:
This filter seems redundant (timewise a bit expensive) as normalisation `make NORM` will kill the redundant axiom?